### ISSUE TITLE
Fontsize extraction error

### ISSUE THREAD

**Issue by Jamiewarb** [dobromir-hristov/tailwindcss-export-config#27]
Seems there is an error when extracting font sizes from Tailwind. 

I believe this is because a fontSize can now take an object as its second argument since 1.7.0. See: https://tailwindcss.com/docs/font-size#providing-a-default-letter-spacing

Please see below:

## Dependencies
```bash
tailwindcss@1.9.6
tailwindcss-export-config@2.3.0
```

## Replication

```js
//tailwind.config.js
module.exports = {
  future: {
    removeDeprecatedGapUtilities: true,
    purgeLayersByDefault: true,
    defaultLineHeights: true,
    standardFontWeights: true,
  },
  prefix: 'u-',
  purge: {
    content: [
      './src/**/*.html',
      './src/**/*.php',
      './src/**/*.js',
      './src/**/*.svg',
      './src/scss/purge-whitelist.scss',
    ],
  },
};
```

```js
const converter = new TailwindExportConfig({
  config: path.resolve('./tailwind.config.js'),
  destination: './_tailwind.config',
  format: 'scss',
  prefix: 'tw',
  flat: false,
  quotedKeys: true,
  preserveKeys: ['colors', 'screens'],
});
converter
  .writeToFile()
  .then(() => {
    console.log('captaincss: Tailwind config exported to', defaultOptions.destination);
  })
  .catch((error) => {
    console.warn('captaincss:', error.message);
  });
```

## Output
```scss
...
$tw-fontSize: (
  "xs": (0.75rem,[object Object]),
  "sm": (0.875rem,[object Object]),
  "base": (1rem,[object Object]),
  "lg": (1.125rem,[object Object]),
  "xl": (1.25rem,[object Object]),
  "2xl": (1.5rem,[object Object]),
  "3xl": (1.875rem,[object Object]),
  "4xl": (2.25rem,[object Object]),
  "5xl": (3rem,[object Object]),
  "6xl": (4rem,[object Object]),
);
...
```

**Comment by dobromir-hristov** [comment]
Hey there, sorry for the late reply. I am pretty busy lately.

Here is what the config looks like.

```js
fontSize: {
      '2xl': ['24px', {
        letterSpacing: '-0.01em',
      }],
      // Or with a default line-height as well
      '3xl': ['32px', {
        letterSpacing: '-0.02em',
        lineHeight: '40px',
      }],
    }
```

How do you think the exported variables should look like, something like this? Would this be even nice to use? 

```scss
$tw-fontSize: (
  "2xl": (24px, (
        letterSpacing: '-0.01em'
       )),
  "3xl": (32px, (
        letterSpacing: '-0.02em',
        lineHeight: '40px',
      )),
```

**Comment by kiril-daskalov** [comment]
I will probably use them as flat so I was thinking what about if they look flat like this: 

```
$fontSize-3xl-letterSpacing: '-0.02em';
$fontSize-3xl-lineHeight: '40px';
```

**Comment by dobromir-hristov** [comment]
I guess it's possible. What about the none flat variants? What i showed makes the most sense, I guess?

**Comment by dobromir-hristov** [comment]
This has been fixed in #46
